scss/_patterns_strip.scss

```diff
@@ -1,9 +1,11 @@
 // Full width strip pattern
 @mixin vf-p-strip {
 
+  // strip
   %strip {
     background-color: transparent;
     clear: both;
+    margin-bottom: 2rem;
```

Comment:
It was an attempt to get around the 'double spacing (margin + padding)' issue when two strips of the same type follow each other - but thinking again, perhaps that a subjective design answer to a question that has not yet been asked. Will change to your suggested approach.